server/jetty-app-custom/src/main/java/io/deephaven/server/custom/CustomAuthorization.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.server.custom;
+
+import io.deephaven.auth.codegen.impl.InputTableServiceContextualAuthWiring;
+import io.deephaven.server.auth.CommunityAuthorizationProvider;
+
+import javax.inject.Inject;
+
+public final class CustomAuthorization extends CommunityAuthorizationProvider {
```

Comment:
Please add javadoc that makes clear what to expect from using this CustomAuthorization implementation.